docs/tools/kompose/user-guide.md

```diff
@@ -14,45 +14,44 @@ redirect_from:
 
 `kompose` is a tool to help users who are familiar with `docker-compose` move to **Kubernetes**. `kompose` takes a Docker Compose file and translates it into Kubernetes resources.
 
+More information about Kompose can be found on the official [http://kompose.io](http://kompose.io/) site.
+
 `kompose` is a convenience tool to go from local Docker development to managing your application with Kubernetes. Transformation of the Docker Compose format to Kubernetes resources manifest may not be exact, but it helps tremendously when first deploying an application on Kubernetes.
 
 ## Use Case
 
 If you have a Docker Compose `docker-compose.yml` or a Docker Distributed Application Bundle `docker-compose-bundle.dab` file, you can convert it into Kubernetes deployments and services like this:
 
 ```console
-$ kompose --bundle docker-compose-bundle.dab convert
-WARN[0000]: Unsupported key networks - ignoring
-file "redis-svc.yaml" created
-file "web-svc.yaml" created
-file "web-deployment.yaml" created
-file "redis-deployment.yaml" created
-
 $ kompose -f docker-compose.yml convert
-WARN[0000]: Unsupported key networks - ignoring
+WARN: Unsupported key networks - ignoring
 file "redis-svc.yaml" created
 file "web-svc.yaml" created
 file "web-deployment.yaml" created
 file "redis-deployment.yaml" created
 ```
 
-## Installation 
+## Installation
+
+We have multiple ways to install Kompose. Our prefered method is downloading the binary from the latest GitHub release.
 
-Grab the latest [release](https://github.com/kubernetes-incubator/kompose/releases) for your OS, untar and extract the binary.
+### GitHub release
 
-### Linux
+Kompose is released via GitHub on a three-week cycle, you can see all current releases on the [GitHub release page](https://github.com/kubernetes-incubator/kompose/releases).
 
 ```sh
-wget https://github.com/kubernetes-incubator/kompose/releases/download/v0.1.2/kompose_linux-amd64.tar.gz
-tar -xvf kompose_linux-amd64.tar.gz --strip 1
-sudo mv kompose /usr/local/bin
-```
+# Linux 
+curl -L https://github.com/kubernetes-incubator/kompose/releases/download/v0.5.0/kompose-linux-amd64 -o kompose
 
-Kompose has support for two providers: OpenShift and Kubernetes.
-You can choose targeted provider either using global option `--provider`, or by setting environment variable `PROVIDER`.
-By setting environment variable `PROVIDER` you can permanently switch to OpenShift provider without need to always specify `--provider openshift` option.
-If no provider is specified Kubernetes is default provider.
+# macOS
+curl -L https://github.com/kubernetes-incubator/kompose/releases/download/v0.5.0/kompose-darwin-amd64 -o kompose
 
+# Windows
+curl -L https://github.com/kubernetes-incubator/kompose/releases/download/v0.5.0/kompose-windows-amd64.exe -o kompose.exe
+
+chmod +x kompose
+sudo mv ./kompose /usr/local/bin/kompose
```

Comment:
@janetkuo I'd be opposed to the chaining of commands since not everyone has sudo / uses /usr/local/bin. 

The way I show chmod +x kompose, etc. is similar to `kubectl` installation instructions: https://kubernetes.io/docs/tasks/kubectl/install/